kubectl-plugin/pkg/util/validation.go

```diff
@@ -21,22 +23,24 @@ func ValidateResourceQuantity(value string, name string) error {
 	return nil
 }
 
-func ValidateTPUNodeSelector(numOfHosts int32, nodeSelector map[string]string) error {
+func ValidateTPU(tpu *string, numOfHosts *int32, nodeSelector map[string]string) error {
```

Comment:
refactored this function to include the logic of checking the TPU string so caller doesn't have to